script.module.youtube.dl/lib/main.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+import sys
+from lib.yd_private_libs import util, servicecontrol, updater
+import xbmc
+import xbmcgui
+
+
+class PlayMonitor(xbmc.Player):
+    def onPlayBackStarted(self):
+        self.setVideoValidity()
```

Comment:
Maybe limit this call to versions below v18 since the event will be caught by onAVStarted?